### ISSUE TITLE
Disable automatic page sync

### ISSUE THREAD

**Issue by w00fz** [trilbymedia/grav-plugin-git-sync#188]
**ard** on discord mentioned the setting being ignored even if disabled and saving still syncing unexpectedly.

**Comment by arifrifai** [comment]
Yes, I did, thanks to raise this issue.

**Comment by AXIM0S** [comment]
And I was thinking it's an issue with my messy setup itself. Glad I'm not alone :joy:
Hoping this bug getting patched soon...

**Comment by w00fz** [comment]
Sorry for the wait, I hope some time next week to be able to get to this issue!

Cheers

**Comment by preraksola** [comment]
I am facing the same issue. I have a pipeline configured to my git repository. On every save a new build is being triggered which is not useful and is consuming resources. Could you please provide an update on this issue? Thanks.

**Comment by w00fz** [comment]
@mahagr can you take a quick look and see if anything stands out? I believe this is related to Flex Pages and 1.7/1.10 , perhaps something not backward compatible is causing this issue.

Cheers

**Comment by mahagr** [comment]
Yeah, we run gitsync manually from flex as it doesn't call the admin event unless it's saving a page.
